indico/web/templates/forms/permissions_widget.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <h3>Permissions</h3>
+    <div id="permissions-widget-{{ field.id }}" class="permissions-widget">
+        {% set event = field.event %}
+        <input type="hidden" value='{{ field._value()|tojson }}' name="permissions">
```

Comment:
`name="{{ field.name }}'`

Reply:
When using `|tojson` outside a script tag, you always need to use `|forceescape` as well